File: core/autocomplete/context/root-path-context/test/files/file1.py
from base_module import BaseClass, Collection, Person, Address
from typing import List

def get_address(person: Person) -> Address:
    return person.address

class Group(BaseClass, Collection):
    def __init__(self, people: List[Person]) -> None:
        super().__init__()
        self.people = people

    def get_person_address(self, person: Person) -> Address:
        return get_address(person)

File: core/autocomplete/context/root-path-context/test/files/file1.ts
import {
  Address,
  Person,
  BaseClass,
  FirstInterface,
  SecondInterface,
  // @ts-ignore
} from "./types";

function getAddress(person: Person): Address {
  return person.address;
}

class Group extends BaseClass implements FirstInterface, SecondInterface {
  people: Person[];

  constructor(people: Person[]) {
    super();
    this.people = people;
  }

  getPersonAddress(person: Person): Address {
    return getAddress(person);
  }
}
